README.md

```diff
@@ -0,0 +1,343 @@
+| :heart: [Contributing](https://github.com/deepset-ai/haystack/#heart-contributing) | We welcome all contributions! |
+
+## Core Features
+
+-   **Latest models**: Utilize all latest transformer based models (BERT, RoBERTa, MiniLM ...) for extractive QA, generative QA and document retrieval.
+-   **Modular**: Multiple choices to fit your tech stack and use case. Pick your favorite database, file converter or modeling framwework.
+-   **Open**: 100% compatible with HuggingFace's model hub. Tight interfaces to other frameworks (Transformers, FARM, sentence-transformers ...)
+-   **Scalable**: Scale to millions of docs via retrievers, production-ready backends like Elasticsearch / FAISS and a fastAPI REST API
+-   **End-to-End**: All tooling in one place: file conversion, cleaning, splitting, training, eval, inference, labeling ...    
+-   **Developer friendly**: Easy to debug, extend and modify.
+-   **Customizable**: Fine-tune models to your own domain or implement your custom DocumentStore.
+-   **Continuous Learning**: Collect new training data via user feedback in production & improve your models continuously
+
+
+## Installation
+
+PyPi:
+
+    pip install farm-haystack
+
+Master branch (if you wanna try the latest features):
+
+    git clone https://github.com/deepset-ai/haystack.git
+    cd haystack
+    pip install --editable .
+
+To update your installation, just do a git pull. The --editable flag
+will update changes immediately.
+
+On Windows you might need:
+```
+pip install farm-haystack -f https://download.pytorch.org/whl/torch_stable.html
+```
+
+## Key Components
+
+![image](https://raw.githubusercontent.com/deepset-ai/haystack/master/docs/_src/img/sketched_concepts_white.png)
+
+1.  **FileConverter**: Extracts pure text from files (pdf, docx, pptx, html ...) 
+2.  **PreProcessor**: Cleans and splits texts into smaller chunks
+3.  **DocumentStore**: Database storing the documents, metadata and vectors for our search. 
+We recommend Elasticsearch or FAISS, but have also more light-weight options for fast prototyping (SQL or In-Memory).
+4.  **Retriever**: Fast algorithms that identify candidate documents for a given query from a large collection of documents. 
+    Retrievers narrow down the search space significantly and are therefore key for scalable QA.
+    Haystack supports sparse methods (TF-IDF, BM25, custom elasticsearch queries)
+    and state of the are dense methods (e.g. sentence-transformers and Dense Passage Retrieval)
+5.  **Reader**: Neural network (e.g. BERT or RoBERTA) that reads through texts in detail
+    to find an answer. The Reader takes multiple passages of text as input and returns top-n answers. Models are trained via [FARM](https://github.com/deepset-ai/FARM) or [Transformers](https://github.com/huggingface/transformers) on SQuAD like tasks.  You can just load a pretrained model from [Hugging Face's model hub](https://huggingface.co/models) or fine-tune it on your own domain data.
```

Comment:
It sounds a little more professional to me to say "you can simply load a pretrained model..."